numba_cuda/numba/cuda/cudadrv/enums.py

```diff
@@ -309,6 +309,9 @@
 # Applicable options: PTX compiler options, ::CU_JIT_FALLBACK_STRATEGY
 CU_JIT_INPUT_LIBRARY = 4
 
+# LTO IR
+CU_JIT_INPUT_LTO_IR = 5
```

Comment:
Technically this should have been `CU_JIT_INPUT_NVVM` as it's the value from the `CUjitInputType_enum` in cuda.h